### ISSUE TITLE
Amalgamation does not include SwapAxis

### ISSUE THREAD

**Issue by d4wud** [dmlc/mxnet#4568]
I'm not sure if this is intended functionality or a bug.  We're hopeful we can do what we need without it using `reshape().` 
```
org.dmlc.mxnet.MxnetException: Failed loading Op swapaxis0 of type SwapAxis: jni/../mxnet_predict-all.cc:25299: Cannot find Operator SwapAxis in registry
at org.dmlc.mxnet.Predictor.createPredictor(Native Method)
at org.dmlc.mxnet.Predictor.<init>(Predictor.java:46)
```
Is there a list of `.so`-supported operations?

**Comment by antinucleon** [comment]
Amalgamation need to be updated. We will include the op while generate amalgamation source code.

**Comment by piiswrong** [comment]
This should be fixed for v0.9.3 and later
